combat-trainer.lic

```diff
@@ -297,7 +299,8 @@ class SetupProcess
   end
 
   def determine_whirlwind_weapon(game_state)
-    offhand_skill = game_state.sort_by_rate_then_rank(game_state.whirlwind_trainables - [game_state.weapon_skill]).first
+    return if game_state.weapon_skill.include?'Twohanded'
```

Comment:
For the mere mortal ruby coders like me, would you be amenable to including the parens around the `include?` method call? Coming from languages where the parens are required, at first I thought this might have been a typo or syntax issue before realizing they're optional. Thanks!
```suggestion
    return if game_state.weapon_skill.include?('Twohanded')
```

Reply:
I won't, honestly I get you, and I did at first (as I came from a C++ background), but the code uses both so often (just below this setting is more and more of it, that I think it's just simply not worth it. It's just all over the place.